src/components/ExperimentalFilters/Row.tsx

```diff
@@ -39,8 +39,8 @@ export const RowComponent = ({
         onChange={(value) => {
           emitter.changeLeftOperator(
             index,
-            value,
-            leftOptions.find((option) => option.value === value.value)?.type
+            value!,
```

Comment:
**Question:** why this change?

Maybe it will be more safe to check if value exist first and then executing `emitter.changeLeftOperator`?